package.json

```diff
@@ -16,7 +16,7 @@
     "postinstall": "husky install",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
-    "test:integration": "jest -c jest.integration.config.ts"
+    "test:integration-dev": "jest --setupFiles ./.integration.test-${ENV}.env -c jest.integration.config.ts"
```

Comment:
Can we put thus back to `test:integration`

Reply:
Also you can set a default value for the environment with `${ENV:-build}`

Reply:
Done.